bigtable-hbase-replication/pom.xml

```diff
@@ -153,11 +153,6 @@
       <version>${project.version}</version>
```

Comment:
Similary, use the project.version for hbase-replication-common in 1.x and 2.x package